lib/src/components/NxSmallThreatCounter/__tests__/NxSmallThreatCounter.test.tsx

```diff
@@ -4,280 +4,176 @@
-  });
+  it('renders a counter with an accessible name of Critical when criticalCount is specified', async function() {
+    const component = quickRender({criticalCount: 0});
 
-  it('renders an .nx-small-threat-counter--unspecified when unspecifiedCount is specified', function() {
-    const component = getMountedComponent({unspecifiedCount: 0});
-    const container = component.find('.nx-small-threat-counter-container');
-    expect(container).toExist();
-    const unspecifiedCounter = component.find('.nx-small-threat-counter--unspecified');
-    expect(unspecifiedCounter.find('.nx-small-threat-counter__count')).toHaveText('0');
-    expect(unspecifiedCounter.find('.nx-small-threat-counter__category')).toHaveText('Unspecified');
-  });
+    await runTimers();
 
-  it('renders all six indicators', function() {
-    const component = getMountedComponent({
-      criticalCount: 66,
-      severeCount: 55,
-      moderateCount: 44,
-      lowCount: 33,
-      noneCount: 22,
-      unspecifiedCount: 42
-    });
-    const criticalCount = component.find('.nx-small-threat-counter--critical'),
-        severeCount = component.find('.nx-small-threat-counter--severe'),
-        moderateCount = component.find('.nx-small-threat-counter--moderate'),
-        lowCount = component.find('.nx-small-threat-counter--low'),
-        noneCount = component.find('.nx-small-threat-counter--none'),
-        unspecifiedCount = component.find('.nx-small-threat-counter--unspecified');
-    expect(criticalCount.find('.nx-small-threat-counter__count')).toHaveText('66');
-    expect(criticalCount.find('.nx-small-threat-counter__category')).toHaveText('Critical');
-    expect(severeCount.find('.nx-small-threat-counter__count')).toHaveText('55');
-    expect(severeCount.find('.nx-small-threat-counter__category')).toHaveText('Severe');
-    expect(moderateCount.find('.nx-small-threat-counter__count')).toHaveText('44');
-    expect(moderateCount.find('.nx-small-threat-counter__category')).toHaveText('Moderate');
-    expect(lowCount.find('.nx-small-threat-counter__count')).toHaveText('33');
-    expect(lowCount.find('.nx-small-threat-counter__category')).toHaveText('Low');
-    expect(noneCount.find('.nx-small-threat-counter__count')).toHaveText('22');
-    expect(noneCount.find('.nx-small-threat-counter__category')).toHaveText('None');
-    expect(unspecifiedCount.find('.nx-small-threat-counter__count')).toHaveText('42');
-    expect(unspecifiedCount.find('.nx-small-threat-counter__category')).toHaveText('Unspecified');
-  });
+    const counter = component.getByTitle('Critical');
 
-  it('correctly assigns supplied class', function() {
-    const classComponent = getMountedComponent({ criticalCount: 0, className: 'test-class' });
-    expect(classComponent.find('.nx-small-threat-counter-container')).toHaveClassName('test-class');
+    expect(counter).toBeInTheDocument();
+    expect(counter).toHaveTextContent('0');
+    expect(counter).toHaveAccessibleName('Critical');
```

Comment:
I actually don't think we need to test the accessible name here.  `<div>`s have a role of `generic`, and the `generic` role prohibits naming.  `title`s are allowed on anything, and [do technically become the name in this case](https://www.w3.org/TR/html-aam-1.0/#section-and-grouping-element-accessible-name-computation), but I don't think any accessibility technology would expect or use it as such here so I don't think we need to test it

Reply:
Agree, updated here dbcb6ce377f41ca58742dba1a29eedf32f170992. Thanks.